web/styles/zulip.css

```diff
@@ -1881,6 +1898,13 @@ div.focused-message-list {
     display: block;
 }
 
+/* In conversation view, always show the recipient bar action buttons */
+div.focused-message-list .recipient_row:only-child {
+    & .recipient_bar_controls {
+        visibility: visible;
+    }
+}
```

Comment:
This check, unfortunately, will be true even if it's not a conversation view, but, say, a "mentions" or "starred" view that happens to only contain one message.

I think you'll need to key this fact off a fact about the view written into the `recipient_row.hbs` template, or maybe applied to `zfilt`. `is-conversation-view` would be a good name for a property added to that. You can add a new first commit that sets this.

And then this could be `div.focused-message-list.is-conversation-view ...`

Reply:
Yes that is true, thank you for spotting this out ! :smile: 

As far as I've understood, I'll have to add some JS logic to check if the view is actually a **_conversation view_**, and if so, then I'll hand over this fact to the `recipient_row.hbs` template and then apply the approach you described.